src/scripts/main.js

```diff
@@ -1,3 +1,7 @@
 'use strict';
 
-// write code here
+const spider = document.querySelector('.spider');
```

Comment:
You should first get the dimensions of the field with the class 'wall' and the dimensions of the spider image, then calculate the centered position.